console/README.md

```diff
@@ -1,24 +1,55 @@
 # RSK Node console
 
-
 ## Setup
 
-Install [NodeJS](https://nodejs.org) and [NPM](https://www.npmjs.com/)
+Install [NodeJS](https://nodejs.org) and [NPM](https://www.npmjs.com/).
+
+From the command line, execute `npm install` to install dependencies.
+
+## Run
+
+To run RSK console in the default host `http://localhost:4444` execute:
+
+```shell
+node console.js
+```
+
+To run RSK console in an specified host execute
 
-From the command line, execute ```npm install``` to install dependencies.
+```shell
+node console.js -server HOST:PORT
+```
 
+To connect via _http_ is not necessary to specify the protocol. Run:
 
-## RUN
+```shell
+node console.js -server "mynode.com"
+```
 
-From the command line, execute ```node console.js -server HOST:PORT```
+To connect via _https_ is necessary to specify the protocol. Run:
+
+```shell
+node console.js -server "https://public-node.rsk.co"
```

Comment:
That exactly line is not working because is trying to connect to `https://public-node.testnet.rsk.co:4444` when needs to connect to `https://public-node.rsk.co:443`